proc-macro/src/types.rs

```diff
@@ -174,7 +174,8 @@ impl Parse for Transparent {
         let lookahead = content.lookahead1();
 
         if lookahead.peek(kw::transparent) {
-            Ok(Self(content.parse::<kw::transparent>()?))
+            content.parse::<kw::transparent>()?;
```

Comment:
I suppose this is fine? I did it to fix a 'field never read' warning.

Reply:
Well with `self.0.span` one could access the span of the custom keyword, which I'd like to retain and rather make clippy shut up. It's not used right now, yet I'd like to retain it.

Reply:
Fixed